server/service.go

```diff
@@ -41,6 +41,21 @@ func respondError(w http.ResponseWriter, code int, message string) {
 	}
 }
 
+func respondOK(w http.ResponseWriter, result any) error {
+	var err error
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	if result == nil {
```

Comment:
instead of passing `nil` you can using an empty struct like `var nilResponse struct{}` and pass `nilResponse` into `respondOK()`. The empty struct will be encoded as `{}\n`, making this function a bit cleaner:
```
if err := json.NewEncoder(w).Encode(result); err != nil {
    return err
}
return nil
```

Reply:
Is it okay to have this `\n` ? If yes, I can make this change.

Reply:
yes, let's just make the proposed change, and get this merged 👍

Reply:
Alright, added in cff01182a553449187c63d803abf0241520c1bb6